### PR TITLE
Added support for missing task schema parameters

### LINKED ISSUE #25: Task schema has missing parameters: assignee ids, user role, est duration, est readiness, attachments
Support for:
- assignee ids (list of assignee ids as list of strings - not required)
- user role (reference already existing UserRole object - not required)
- estimated duration (not required)
- estimated readiness (not required)
- attachment urls (not required) / or attachments in general where each attachment would have: type, name and url

### PR DESCRIPTION
**Pull request by Daniilmass**
Resolves #25

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -1355,6 +1355,26 @@ components:
...
+          description: Estimated duration of the task
+        estimatedReadiness:
+          type: string
+          description: Estimated readiness of the task
+        attachmentUrls:
+          type: array
+          items:
+            type: string
+            format: url
+          description: URLs of attachments related to the task
```
Comment (Daniilmass) [inline swagger.yaml:1377]: I am thinking wether this is enough or should we create a separate attachment object? I incline to keeping it like that for simplicity sake... @kaeline what do you think?

Comment (kaeline) [reply swagger.yaml:1377]: urls should be ok since those are just links after all

File: swagger.yaml [hunk 2]
```diff
@@ -1355,6 +1355,26 @@ components:
...
+          description: List of assignee user IDs
+        userRole:
+          $ref: '#/components/schemas/UserRole'
+          description: User role suggested for the task
+        estimatedDuration:
+          type: string
+          description: Estimated duration of the task
+        estimatedReadiness:
+          type: string
+          description: Estimated readiness of the task
```
Comment (kaeline) [inline swagger.yaml:1371]: This does not depend on task start and end dates, right?

**Review by kaeline — APPROVED** [review]

File: swagger.yaml (hunk 2 above)
Comment (Daniilmass) [reply swagger.yaml:1371]: No, those are just user defined valuse, so probably string is good enough

**Pull request merged**